src/lib/utils/sqlite_add_indices.cpp

```diff
@@ -36,12 +36,17 @@ void add_indices_to_sqlite(const std::string& schema_file_path, const std::strin
 
   // Recreate tables using the passed schema sql file
   std::ifstream schema_file(schema_file_path);
+  Assert(schema_file.good(), std::string{"Schema file "} + schema_file_path +
+                                 " not found - are you calling the binary from a different working directory?");
```

Comment:
Do we need to call from the binary root? 

“Different work dir” is not clear to me

Reply:
Doesn't really matter, as long as the file exists. This could be the root dir or the build dir.

I believe that the full message is clearer, but maybe you have a better idea:

> libc++abi.dylib: terminating with uncaught exception of type std::logic_error: src/lib/utils/sqlite_add_indices.cpp:40 Schema file resources/benchmark/tpch/schema.sql not found - are you calling the binary from a different working directory?

Reply:
Even this confuses me. The working directory is the one I am currently in. Can I be in another working directory? Or is that supposed to also cover cases of sub shells etc.?

Maybe we can directly say, that we cannot access the `resources` directory which we expect to be in the "working directory"?

Reply:
Mmh, but this file does not really know anything about `resources/`.

Maybe "try running the binary from the Hyrise root or the build directory"?

Reply:
(btw, *working directory* is a POSIX term: https://www.gnu.org/software/libc/manual/html_node/Working-Directory.html)